### ISSUE TITLE
Messages not send to SUBmcz

### ISSUE THREAD

**Issue by tharts** [Chibald/maestrogateway#3]
Installed the script like described in the README.
When starting the daemon service using
`sudo systemctl start maestro.service`
it does not return to command line until CTRL-C is pressed.

Checking the status with
`sudo systemctl status maestro.service`
gives:

`● maestro.service - MCZ Maestro Equipment Gateway with MQTT Server
   Loaded: loaded (/etc/systemd/system/maestro.service; disabled; vendor preset: enabled)
   Active: activating (start) since Wed 2020-01-29 22:42:47 CET; 2min 25s ago
Cntrl PID: 10051 (python)
    Tasks: 5 (limit: 4915)
   Memory: 7.4M
   CGroup: /system.slice/maestro.service
           └─10051 /usr/bin/python /opt/maestro/maestro.py

Jan 29 22:42:47 openhab systemd[1]: Starting MCZ Maestro Equipment Gateway with MQTT Server...`

It looks like the service doesn't start completely.

Subscribing the channel PUBmcz works and status messages from the stove are received.
Publishing commands to the SUBmcz topic does nothing. No reaction from the stove.

**Comment by Chibald** [comment]
about systemctl not returning; it seems there is an error in the service definition file that causes this. since it just works (although not returning to command prompt) I didn't bother to look into it tbh. 

But the output put you pasted shows the service is running actually, you also get messages from the stove so thats good. 

Whats inside the activity.log file?

**Comment by tharts** [comment]
I got it working now. 
Somehow the script lost connection to the websocket for sending commands. Since there were incomming messages I thought it was working.
In OH2 I uninstalled the embedded broker and switched to Mosquitto which I'm using with the new OH MQTT-Binding.
Nice script :-) Thanks for providing it to us.
Now I will try to figure out how to control it from Google Home (on/off of stove already works, just need to get the thermostat thing up'n running to set and see the temperatures.
